sql/core/src/test/resources/sql-tests/analyzer-results/double-quoted-identifiers-enabled.sql.out

```diff
@@ -154,6 +154,215 @@ org.apache.spark.sql.AnalysisException
 }
 
 
+-- !query
+select 1 from "not_""exists"
```

Comment:
This PR updates the string literal parsing. We don't need to test so many different SQL statements that contain strig literal, but simply `SELECT string literal` and focus more on the string literal itself
```
SELECT 'a''b'
SELECT 'a' 'b'
...
```

Reply:
It's indeed overkill to test these double-quoted identifiers here so I removed most of the unrelated tests, but I still remained `select 1 from "not_""exists"` since this PR also changed the behavior when handling double-quoted identifiers.